test/com/core/tictactoe/BoardTests.java

```diff
@@ -0,0 +1,171 @@
+package com.core.tictactoe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+
+public class BoardTests {
+
+    private Board board;
+    private Player player;
+    private Player player2;
+
+    @BeforeEach
+    void instantiateBoard() {
+        board = new Board();
+        player = new Player("X");
+        player2 = new Player("Y");
+    }
+
+    @Test
+    void BoardIsCreatedWithPlacesArray() {
+        assertThat(board.places,isA(ArrayList.class));
+    }
+
+    @Test
+    void BoardHasFilledPlacesWhenCreated() {
```

Comment:
what does 'filled' mean ?  To me I thought it meant with a 'mark' in it, but this is the opposite.  
What is the board when it doesn't have marks?  Empty?

Reply:
Initially the board.places is an ArrayList with integers from 0 to 8. During the game the integers are changed to players marks. I've changed the naming to be more accurate.